src/sentry/auth/providers/fly/provider.py

```diff
@@ -0,0 +1,8 @@
+from sentry.auth.partnership_config import SPONSOR_OAUTH_NAME, ChannelName
+from sentry.auth.providers.oauth2 import OAuth2Provider
+
+
+class FlyOAuth2Provider(OAuth2Provider):
```

Comment:
```suggestion
class FlyIOAuth2Provider(OAuth2Provider):
```

Typo in the class name -- missing an `i`.

Reply:
I actually didn't mean to use the IO is naming. Then it would be like FlyIOOAuth2Provider

Reply:
🤦 I totally see that now. My bad. 😅